### ISSUE TITLE
Image Hosting Support

### ISSUE THREAD

**Issue by bmonkman** [commitdev/zero#234]
Add support for hosting images
Required:
 - Upload
 - Download

Nice to have:
 - Processing
 - Signed download URLS

We already have Go code for some of this in other projects, talk to Bill.

Most likely this will touch backend, frontend, infra.

**Comment by GrooveStomp** [comment]
@bmonkman you have existing Go code for some of this?
I'd like to start breaking this down into implementable pieces.

**Comment by GrooveStomp** [comment]
@bmonkman The basic functionality to support this is now delivered for zero, but we're missing image processing.
Should we keep the epic alive for that?

**Comment by bmonkman** [comment]
I don't think so, as we don't have clear requirements or need for that right now. We can just add that to the backlog as a feature to be considered later.
